src/swap/core/getParamsForToken.ts

```diff
@@ -1,15 +1,17 @@
 import { formatDecimals } from './formatDecimals';
-import type { GetQuoteParams, GetQuoteAPIParams } from '../types';
+import type { SwapParams, SwapAPIParams, GetSwapParams } from '../types';
 
 /**
  * Converts parameters with `Token` to ones with address. Additionally adds default values for optional request fields.
  */
-export function getParamsForToken(params: GetQuoteParams): GetQuoteAPIParams {
+export function getParamsForToken(params: SwapParams): SwapAPIParams {
   const { from, to, amount, amountReference, isAmountInDecimals } = params;
+  const { fromAddress } = params as GetSwapParams;
```

Comment:
Can we add more comments here, I am a bit confused. Like who needs `fromAddress` and how they use it. And when is not there, will just be empty?